openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/management/jvmcpumonitorbean/ThreadCategory.java

```diff
@@ -1,132 +1,132 @@
+					long startTime = System.nanoTime();
+					long busyTime = 1500 * 1000000L; // 1.5 secs
+					while ((System.nanoTime() - startTime) < busyTime) {
+						mxbean.setThreadCategory(threadId, newCategory);
+					}
+				}
+			} else {
+				threadId = Thread.currentThread().getId();
+				try {
+					mxbean.setThreadCategory(threadId, newCategory);
+					
+					if( (mxbean.getThreadCategory(threadId)).equalsIgnoreCase("Application-User0") ) {
+						System.err.println("Setting to Application-User0 category is expected to fail..");
+						return;
+					}
+				} catch(IllegalArgumentException iae) {
+					if(newCategory.equalsIgnoreCase("Application-User0")) {
+						mxbean.setThreadCategory(threadId, "Application");
+					}
+					else {
+						iae.printStackTrace();
+						return;
+					}
+				}
+				
+				Random rand = new Random();
+				@SuppressWarnings("unused")
+				double result = 0;
+								
+				long startTime = System.nanoTime();
+				long busyTime = 1000 * 1000000L;
+				
+				if(newCategory.equalsIgnoreCase("Application-User1")) {
+					while ((System.nanoTime() - startTime) < busyTime) {
+						double value = rand.nextDouble();
+						result = java.lang.Math.log(value);
+					}
+					mxbean.setThreadCategory(threadId, "Application");
+				} else if(newCategory.equalsIgnoreCase("Application-User2")) {
+					while ((System.nanoTime() - startTime) < busyTime) {	
+						double value = rand.nextDouble();
+						result = java.lang.Math.cos(value);
+					}
+					mxbean.setThreadCategory(threadId, "Application");
+				} else if(newCategory.equalsIgnoreCase("Application-User3")) {
+					while ((System.nanoTime() - startTime) < busyTime) {
+						double value = rand.nextDouble();
+						result = java.lang.Math.tan(value);
+					}
+					mxbean.setThreadCategory(threadId, "Application");
+				} else if(newCategory.equalsIgnoreCase("Application-User4")) {
+					while ((System.nanoTime() - startTime) < busyTime) {
+						double value = rand.nextDouble();
+						result = java.lang.Math.sin(value);
+					}
+					mxbean.setThreadCategory(threadId, "Application");
+				} else if(newCategory.equalsIgnoreCase("Application-User5")) {
+					while ((System.nanoTime() - startTime) < busyTime) {
+						double value = rand.nextDouble();
+						result = java.lang.Math.exp(value);
+					}
+					mxbean.setThreadCategory(threadId, "Application");
+				} else if(newCategory.equalsIgnoreCase("Resource-Monitor")) {
+					while ( !isInterrupted ) {
+						JvmCpuMonitorInfo jcmInfo = mxbean.getThreadsCpuUsage(new JvmCpuMonitorInfo());
+						@SuppressWarnings("unused")
+						long appCpuTime = jcmInfo.getApplicationCpuTime();
+					}		
+				} else {			
+					while ((System.nanoTime() - startTime) < busyTime) {
+						double value = rand.nextDouble();
+						result = java.lang.Math.exp(value);
+					}
+				}
+			}
+		} catch(Exception e) {
+			e.printStackTrace();
+		}
+	}
+	
+	public void setResourceMonitorInterruptedStatus(Boolean b) {
+		isInterrupted = b;
+	}
+}
+
```

Comment:
A number of the test.jlm files had Windows CRLF rather than unix line feed so I fixed them - but that caused the PR checker to insist on updating the copyright year.